### DISCUSSION TITLE
iOS Slow Motion Videos not "slow" on immich

### DISCUSSION THREAD

**Discussion by mirisbowring** [immich-app/immich#3321] [Q&A]
Hi,

i have some Videos on my iPhone that have been recorded via the "Slow-Mo" Mode of the iPhone Camera.
In my Apple Gallery, the Video is playing in slow motion.

After uploading this video to immich, the slow motion effect is not in place anymore.

Do i need to configure something or is this currently not supported?

Thanks!

**Comment by alextran1502** [comment]
Hello, this is currently not supported yet. I believe it is proprietary format from Apple

**Reply by HippoBloke** [reply]
Just bumping this again - is it on the roadmap. I notice Synology Photos can play the slow motion ?

**Comment by ddshd** [comment]
Slow motion video on iOS is just high fps video. The slow motion effect is done by Apple. The app just uploads the high fps video as is.

**Reply by mirisbowring** [reply]
How does apple detect then that the video is slow mo? Is there a metadate field?

In this case, Immich should only need to slow down playback speed right?

**Reply by jimmyjules153** [reply]
bump 😉 I would also love this feature, it sucks to have to download high FPS videos and play them with VLC to slow down the playback speed. If Immich could detect the framerate of the video and offer slow-mo as a playback option where it just decreases the playback speed that'd surely be a quick and clean-ish fix..! 🤔 
At the end of the day this is a small problem. I still love this app and everything the Immich team are doing!💜

**Reply by coryrearick9900** [reply]
I did find a bit of a janky solution.  It seems like you can have it export the slow-mo video with slow-mo intact if you _crop the video_ from the iOS photos app.

I was having the issue of trying to have slow-mo videos appear instact on my desktop client but it wasn't working, they were playing at full speed all the time.  What I did was edited one of them, cropped it ever so slightly, and then just to be safe I trimmed the length of the video to cut off a very small amount that way my phone gave me the option to either "Save Video" or "Save Video as New Clip", which I chose to save it as a new clip just to be safe.  After this new clip got synced to my desktop client the slow-mo seemed to play intact.

What I did:
1) Take existing slow-mo video
2) Crop the video ever so slightly to trim out as little as possible
3) Trim out a very little amount of the video from the end to force the phone to allow me to save the video as a new clip
4) Synced this to my Immich server
5) Slow-mo seemed to play intact.

I hope this helps someone!  :)

**Comment by jimmyjules153** [comment]
Heyyyy this is extraordinarily helpful!!
Thank you so much for the tip 💜

**Comment by Cheekie25** [comment]
This would be a good feature to add !

**Comment by AngelaDMerkel** [comment]
This can be accomplished with ffmpeg using a command for a speed ramp like: `ffmpeg -y -i 15.mp4 -filter:v "setpts=(1.5-0.002*N)*PTS"` but this requires identifying the `UITrackSegment` and then the using that to determine whether the video is slowmo. There is a good discussion of this contained in this [blog post](https://medium.com/@xcadaverx/avfoundation-cmtimemapping-c3521d4da6b1).

**Comment by blue-kaleidoscope** [comment]
Is this now supported on purpose? I did not see anything in the release notes (or I oversaw it) but with v1.119 a recently uploaded iPhone SloMo video is correctly visualized when using the browser. I'm amazed!

**Comment by HippoBloke** [comment]
Good spot. I reuploaded an old one and it is now working correctly.

**Comment by mertalev** [comment]
Closing since this is apparently fixed (and definitely fixed with the native player)

**Reply by timaschew** [reply]
For me it's not working with 1.131.3, slowmotion videos are not played slowly, just normal.
Both, original and converted video are played in slowmotion if I open them in QuickTime Player.

**Reply by kattkaries** [reply]
+1
